webapp/src/Controller/API/LanguageController.php

```diff
@@ -122,6 +123,84 @@ public function updateExecutableActions(Request $request, string $id): void
         $this->dj->auditlog('executable', $language->getLangid(), 'updated');
     }
 
+    #[IsGranted('ROLE_ADMIN')]
```

Comment:
Do we want this to be API_WRITER? We do for other endpoints but as this is setup of the instance I can see why it can be ADMIN only.

Reply:
Looking at other places, I think this is fine as is, e.g.:
- https://github.com/DOMjudge/domjudge/blob/main/webapp/src/Controller/API/ContestController.php#L68
- https://github.com/DOMjudge/domjudge/blob/main/webapp/src/Controller/API/ProblemController.php#L57
- https://github.com/DOMjudge/domjudge/blob/main/webapp/src/Controller/API/GeneralInfoController.php#L244